test.acceptance.sh

```diff
@@ -2,5 +2,5 @@
 
 source ./test.common.sh
 
-go test ./test/acceptance -count 100 -timeout 6m -failfast > test.out
+STANDALONE=true go test ./test/acceptance -count 100 -timeout 6m -failfast > test.out
```

Comment:
`STANDALONE` is a very confusing name for this env variable. 
Maybe rename it to `NO_LOG_STDOUT`? It's a much clearer name that says exactly what it does..

Reply:
Fixed